app/components/primer/dropdown/menu_component.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Primer
+  module Dropdown
+    # DropdownMenus are lightweight context menus for housing navigation and actions.
+    # They're great for instances where you don't need the full power (and code)
+    # of the select menu.
+    class MenuComponent < Primer::Component
+      include ViewComponent::SlotableV2
+
+      SCHEME_DEFAULT = :default
+      SCHEME_MAPPINGS = {
+        SCHEME_DEFAULT => "",
+        :dark => "dropdown-menu-dark"
+      }.freeze
+
+      DIRECTION_DEFAULT = :se
+      DIRECTION_OPTIONS = [DIRECTION_DEFAULT, :sw, :w, :e, :ne, :s].freeze
+
+      renders_many :items, lambda { |divider: false, **item_arguments|
```

Comment:
It's a bit of a nitpick, but what do you think about _always_ calling system arguments as such? (vs. `item_arguments` here)